src/store/ChartState.js

```diff
@@ -105,33 +107,35 @@ class ChartState {
     }
 
     saveDrawings() {
+        if (!this.chartId) return;
         const obj = this.stxx.exportDrawings();
         const symbol = this.stxx.chart.symbol;
         if (obj.length === 0) {
-            CIQ.localStorage.removeItem(symbol);
+            CIQ.localStorage.removeItem(`${this.chartId}-${symbol}`);
         } else {
-            CIQ.localStorageSetItem(symbol, JSON.stringify(obj));
+            CIQ.localStorageSetItem(`${this.chartId}-${symbol}`, JSON.stringify(obj));
         }
     }
 
     restoreDrawings() {
-        const drawings = createObjectFromLocalStorage(this.stxx.chart.symbol);
+        const drawings = createObjectFromLocalStorage(`${this.chartId}-${this.stxx.chart.symbol}`);
         if (drawings) {
             this.stxx.importDrawings(drawings);
             this.stxx.draw();
         }
     }
 
     restorePreferences() {
-        const pref = createObjectFromLocalStorage(`preferences-${this.id}`);
+        const pref = createObjectFromLocalStorage(`${this.chartId}-preferences-${this.id}`);
```

Comment:
@brucebinary what about here. i didnt know what its used for so i didnt remove it.

Reply:
me neither. Have a look at the docs and see what it does? If it doesn't save anything then just remove it. This is from the demo code

Reply:
@brucebinary  Im gonna remove it.Yes this never appears in local storage. i dont think it has any use.